Formula/jfrog-cli-go.rb

```diff
@@ -14,7 +14,6 @@ class JfrogCliGo < Formula
   depends_on "go" => :build
 
   def install
-    system "go", "run", "./python/addresources.go"
```

Comment:
This step got removed in this commit, https://github.com/jfrog/jfrog-cli/commit/94dd4c1f4287a1c5b645f332dd235a83001f4acf#diff-0b83f9dedf40d7356e5ca147a077acb4